app/views/users/_friendship.html.erb

```diff
@@ -0,0 +1,26 @@
+<% if @recieved.include?(user) && user.id != current_user.id %>
+  <%= form_with(url: friendships_accept_path, method: 'post', local: true, class: "d-inline") do |f| %>
+    <%= hidden_field(:friendship, :user_id, value: user.id) %>
+    <span class="actions">
+      <%= f.submit "Accept Request", class: "btn btn-primary" %>
+    </span>
+  <% end %>
+  <%= form_with(url: friendships_reject_path, method: 'post', local: true, class: "d-inline") do |f| %>
+    <%= hidden_field(:friendship, :user_id, value: user.id) %>
+    <span class="actions">
+      <%= f.submit "Reject Request", class: "btn btn-primary" %>
+    </span>
+  <% end %>
+<% elsif @pending.include?(user) && user.id != current_user.id %>
```

Comment:
We can reuse `is_present?` method we wrote above :point_up:  in oder to remove all the logic from the views.